Updated notebook openai_functions_agent.ipynb

sumanthdonthula opened the pull request:
Here's the updated code with grammatical improvements, added comments, and placeholders for users to add their API keys and customize the notebook for their specific use cases.

Linked to issue #6178

sumanthdonthula committed:
Added Comments

sumanthdonthula committed:
Update openai_functions_agent.ipynb

Here's the updated code with grammatical improvements, added comments, and placeholders for users to add their API keys and customize the notebook for their specific use cases.

eyurtsev commented:
@sumanthdonthula thanks for the contribution! Looking good! 

I'm wondering if we could paraphrase the "train" word in this sentence:

"The agent will be trained to perform 3 search functionalities with the respective tools"

There is no training in the machine learning sense. Maybe something alone the lines that  "the agent is given access to 3 tools or something of that nature"

sumanthdonthula committed:
Update openai_functions_agent.ipynb

Changed the word "Train" in description

The agent is given ability to perform 3 search functionalities with the respective tools

sumanthdonthula commented:
Hi @eyurtsev,

Sorry, made changes. Thanks for the suggestion. Now the sentence looks like, 

The agent is given ability to perform 3 search functionalities with the respective tools.

sumanthdonthula commented:
@eyurtsev can you please validate the code now and approve the PR.
